src/commands/login.ts

```diff
@@ -48,8 +56,15 @@ export const login = async (
   await clearIdentityCache();
   await writeIdentity(orgWithSlug, tokenResponse);
 
+  if (orgData.config) {
+    await saveTenantConfig(orgData.config);
+  }
```

Comment:
Doesn't this have to move up, right after `if (!orgData) throw "Could not find organization";`?

Afaict currently we will log the user in to the bootstrap project and whatever the org is configured for login there. I think for orgs that actually use a different host, the login information is going to be in the Firestore of that other host (= P0 environment), and the p0-prod document for their org will only contain a config that points to the right environment.

So we don't want to log these users in to our p0-prod. We want them to log in to their environment. When `loginFn` is called tenantConfig should already be updated with the right config.

Reply:
A second thought: we could always save tenantConfig and use the bootstrapConfig as a fallback right here. Then we don't have to pre-assign in `drivers/config.ts`, tenantConfig can be undefined until we save here. 
```
await saveTenantConfig(orgData.config ?? bootstrapConfig);
```
Or, we could also read tenantConfig lazily in a `getTenantConfig` function that checks the variable, then checks the file, then loads. Since then you can check `if tenantConfig === undefined`.

Reply:
Done